client/src/components/Utilities/GetSequenceIds/GetSequenceIds.tsx

```diff
@@ -1,22 +1,150 @@
-import { TextField } from "@material-ui/core";
-import { useState } from "react";
+import {
+  Button,
+  Link,
+  makeStyles,
+  TextField,
+  Typography,
+} from "@material-ui/core";
+import React, { useEffect, useState } from "react";
 import "./GetSequenceIds.scss";
+import { getSequenceIds } from "../../../services/main";
+import { useColorTheme } from "../../../global/contexts/Theme/ColorThemeContext";
 
 const GetSequenceIds: React.FC = () => {
   const [inputSequence, setInputSequence] = useState<string>("");
+  const [helperText, setHelperText] = useState<string>("");
+  const [refseqId, setRefseqId] = useState<string>("");
+  const [ga4ghId, setGa4ghId] = useState<string>("");
+  const [aliases, setAliases] = useState<string[]>([]);
+
+  useEffect(() => {
+    if (inputSequence) {
+      getSequenceIds(inputSequence).then((sequenceIdsResponse) => {
+        if (sequenceIdsResponse.warnings) {
+          setHelperText("Unrecognized sequence");
+          setRefseqId("");
+          setGa4ghId("");
+          setAliases([]);
```

Comment:
@katiestahl is this the right way to handle this? Am i going to run awry of those concurrency conditions from the gene autocomplete field?

Reply:
I think this should be okay hopefully. The issue with the last concurrency problem was that there were nested axios requests that were asynchronous but relied on each other